Documentation/ApiOverview/Fal/Administration/Maintenance.rst

```diff
@@ -60,3 +60,75 @@ Here you can choose to delete all files in :file:`fileadmin/_processed_/`
 
 This cleanup is also good if processed files have accumulated for a
 long time. Many of them may then be obsolete.
+
+.. attention::
+
+    If you delete processed files, you should flush the (page) cache immediately
+    afterwards. If pages are cached and the page uses processed images, these
+    will not be regenerated on the fly when a page is loaded. Ideally, make sure
+    the removal of the processed files and flushing of page cache is one atomic
+    operation which is performed as quickly as possible.
+
+Also, deleting processed files while editors are active is not ideal.
+Preferably, lock the TYPO3 Backend before you remove the processed files.
+
+See also the next section in case you have many processed files and removal
+would take long (e.g. several minutes).
+
+Bulk removal of processed files
+-------------------------------
+
+This entire section is a "use at your own risk" hint.
+
+In general remove processed files as one atomic operation together with flushing
+the page cache afterwards.
+
+Removing processed files may take long if there are many. As a shortcut, you may
+remove all at once, but always be sure to consider the database table
+(sys_file_processedfiles) **and** the files on the filesystem (e.g.
+file:`fileadmin/_processed_`).
+
+TYPO3 uses storage 0 as default storage if files are not found in one of
```

Comment:
should we link to / explain what "storage 0" means?

Reply:
> should we link to / explain what "storage 0" means?

That is a good idea.

The only place I found where this is explained:
https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Fal/UsingFal/ExamplesFileFolder.html

I don't think this is a great place to link to because there is no dedicated section for this (and not the best place to explain storage 0). It might make sense to add this here: https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Fal/Architecture/Database.html#sys-file-storage or here: https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Fal/Concepts/Index.html#storages-and-drivers

I would like to address that in separate PR because there is also a fuzziness in some places what is meant by "default storage" (might be uid=0 with undefined storage in sys_file_storage or it might be whatever is defined as storage in sys_file_storage, usually 1 = fileadmin and returned via `$this->storageRepository->getDefaultStorage()`).

Reply:
section is now removed.